host/src/gatt.rs

```diff
@@ -60,8 +115,9 @@ impl<'stack> GattData<'stack> {
 
     /// Respond directly to request.
     pub async fn reply(self, rsp: AttRsp<'_>) -> Result<(), Error> {
-        let pdu = send(&self.connection, AttServer::Response(rsp))?;
-        self.connection.send(pdu).await;
+        let connection = self.connection.raw();
+        let pdu = send(connection, AttServer::Response(rsp))?;
+        connection.send(pdu).await;
```

Comment:
Now that you have to create a GattConnection, and therefore an AttributeServer, this API is kind of pointless, because the purpose of this was that you didn't have to create an AttributeServer in order to use it.